### PR TITLE
Fix incorrect information in schema change + prepared statement limitations

### PR DESCRIPTION
**Pull request by mgartner**

### CODE REVIEW

File: src/current/_includes/v24.1/known-limitations/schema-changes-between-prepared-statements.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 When the schema of a table targeted by a prepared statement changes before the prepared statement is executed, CockroachDB allows the prepared statement to return results based on the changed table schema, for example:
 
+When object definitions referenced in a prepared statement are altered the prepared statement is created, future executions of the prepared statement could result in an error. For example, adding a column to a table referenced in a prepared statement with a `SELECT *` clause will result in an error:
```
Comment (taroface) [inline src/current/_includes/v24.1/known-limitations/schema-changes-between-prepared-statements.md:3]: It looks like there is a missing word. Is this correct?

```suggestion
When object definitions referenced in a prepared statement are altered the after prepared statement is created, future executions of the prepared statement could result in an error. For example, adding a column to a table referenced in a prepared statement with a `SELECT *` clause will result in an error:
```

**Review by taroface — COMMENTED** [review]
@mgartner Thanks for filing this. Was this section always incorrect (should this PR technically apply to all CockroachDB versions), or did it become incorrect at a specific version?

Also have 1 in-line question. I can take over the PR since we probably need to update a few additional docs versions.

File: src/current/_includes/v24.1/known-limitations/schema-changes-between-prepared-statements.md (hunk 1 above)
Comment (mgartner) [reply src/current/_includes/v24.1/known-limitations/schema-changes-between-prepared-statements.md:3]: Good catch. LGTM

**Comment by mgartner** [comment]
> Was this section always incorrect (should this PR technically apply to all CockroachDB versions), or did it become incorrect at a specific version?

Yes, I believe this has always been incorrect.

**Review by taroface — APPROVED** [review]

**Pull request merged**